sdk/python/kfp/cli/cli.py

```diff
@@ -52,5 +52,5 @@ def main():
     try:
         cli(obj={}, auto_envvar_prefix='KFP')
     except Exception as e:
-        logging.error(e)
+        click.secho(str(e), fg='red', err=True)
```

Comment:
Is it your intention to also change the error handling in this case? As I understand it this will also through an error compare to the current implementation?

Reply:
Yes, it is my intention to change the logging for the error message with a red color highlighted message text.
This will also work similarly to `logging.error` by` writing the error to `stderr`.

Reply:
@NikeNano, let me know if I understood your question wrong.

Reply:
You are correct, I read the docs a bit to quickly for click.secho, my bad!

Reply:
@NikeNano, would you be able to trigger test if everything looks good to you? I re-requested a review from you previously.